modules/missenaBidAdapter.js

```diff
@@ -0,0 +1,86 @@
+import * as utils from '../src/utils.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+
+const BIDDER_CODE = 'missena';
+const ENDPOINT_URL = 'https://bid.missena.io/';
+
+export const spec = {
```

Comment:
can you please add your gvlid if you have one?

Reply:
Thanks! I've added our global vendor ID to the spec.